database.hh

```diff
@@ -883,6 +883,7 @@ public:
     void try_trigger_compaction() noexcept;
     future<> run_compaction(sstables::compaction_descriptor descriptor);
     void trigger_offstrategy_compaction();
+    future<> submit_offstrategy_compaction();
```

Comment:
nitpick: It's not great that we have a string of similarly-named functions {trigger,submit,run}_offstrategy_compaction()... Nobody will ever remember what's the difference between all those functions.

Reply:
Fixed in v4.